melodic/distribution.yaml

```diff
@@ -5221,6 +5221,17 @@ repositories:
       url: https://github.com/ros-planning/panda_moveit_config.git
       version: melodic-devel
     status: maintained
+  parameter_assertions:
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/RoboJackets/rj-ros-common-release.git
+      version: 0.0.0-1
```

Comment:
Oh, also, a version of 0.0.0 is highly unusual.  I'd suggest at least `0.1.0`.

Reply:
Do I need to re-release to fix this?

Reply:
Yeah, you'd have to run `catkin_prepare_release --bump minor`, follow the instructions, then do another `bloom-release`.  There's a tutorial on how to do it: http://wiki.ros.org/bloom/Tutorials/ReleaseCatkinPackage

Reply:
Alright I'm closing this one since `bloom-release` made a PR automatically [here](https://github.com/ros/rosdistro/pull/21986)